docs/book/src/tasks/experimental-features/cluster-class/operate-cluster.md

```diff
@@ -42,3 +52,96 @@ kubeadmcontrolplane.controlplane.cluster.x-k8s.io/clusterclass-quickstart-XXXX
 NAME                                                                             CLUSTER                   REPLICAS   READY   UPDATED   UNAVAILABLE   PHASE     AGE     VERSION
 machinedeployment.cluster.x-k8s.io/clusterclass-quickstart-linux-workers-XXXX    clusterclass-quickstart   1          1       1         0             Running   7m29s   v1.22.0
 ```
+
+## Scale a MachineDeployment
+Assume we have created a CAPD cluster with ClusterClass and Kubernetes v1.23.3 (as documented in the [Quick Start guide]). Initially we should have a MachineDeployment with 3 replicas. Running
+```bash 
+kubectl get machinedeployments
+```
+Will give us:
+```bash 
+NAME                                                            CLUSTER           REPLICAS   READY   UPDATED   UNAVAILABLE   PHASE     AGE   VERSION
+machinedeployment.cluster.x-k8s.io/capi-quickstart-md-0-XXXX   capi-quickstart   3          3       3         0             Running   21m   v1.23.3
+```
```

Comment:
```suggestion
Assume we have created a CAPD cluster with ClusterClass and Kubernetes v1.23.3 (as documented in the [Quick Start guide]). Initially we should have a MachineDeployment with 3 replicas.
```bash 
$ kubectl get machinedeployments
NAME                                                            CLUSTER           REPLICAS   READY   UPDATED   UNAVAILABLE   PHASE     AGE   VERSION
machinedeployment.cluster.x-k8s.io/capi-quickstart-md-0-XXXX   capi-quickstart   3          3       3         0             Running   21m   v1.23.3
```
```

I think you can simplify like this, if you want

Reply:
I'm fine either way if we pick a pattern and stick to it - this is the sort of style we use in the quickstart so I think if we change it here we should change it there and see where else it might be better.

Reply:
Up to you. I would slightly prefer having just one block vs. two as it just seems more concise to me .

Reply:
I'm going to resolve this, but I think for following along having the commands in separate blocks is better like in the start guide. I don't think it adds confusion to have these mixed up, but maybe there's a way, using backticks or something in MD, to make the difference between command and output clearer. I'll create a follow up issue.